js/api/Surveys.js

```diff
@@ -216,16 +226,17 @@ export function loadSurveys(callback) {
 /**
  * fetches remote data for surveys and invitations
  *
+ * @param {object}   options, options object to be passed to loadSurveys
  * @param {function} done, the callback for when the async operations are done
  */
-export function loadSurveyList(done) {
+export function loadSurveyList(options = {}, done) {
   async.auto({
-    surveys: (cb) => {
-      loadSurveys(cb);
-    },
     invitations: (cb) => {
       loadInvitations(cb);
     },
+    surveys: ['invitations', (cb) => {
```

Comment:
We're loading Triggers/Questions of accepted Invitations on `loadSurveys`. This can create a race condition in which triggers aren't loaded on the first install, causing some of the errors we were experiencing with Surveys missing their form data.